charts/gha-runner-scale-set-controller/templates/deployment.yaml

```diff
@@ -68,6 +68,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: CONTROLLER_MANAGER_LISTENER_IMAGE_PULL_POLICY
```

Comment:
do you want to add an L0 test for the chart?

Reply:
Adding it, good call :smile: 
We have a mechanism to ignore value if it is incorrect or not set, but you are right!